libs/sql-parser/src/main/java/io/crate/sql/SqlFormatter.java

```diff
@@ -236,6 +238,33 @@ public Void visitCreateServer(CreateServer createServer, Integer indent) {
             return null;
         }
 
+        @Override
+        public Void visitCreateSchema(CreateSchema createSchema, Integer indent) {
+            append(indent, "CREATE SCHEMA ");
+            if (createSchema.ifNotExists()) {
+                append(indent, "IF NOT EXISTS ");
+            }
+            append(indent, createSchema.name());
```

Comment:
shouldn't the name be quoted if needed?
```suggestion
            append(indent, quoteIdentifierIfNeeded(createSchema.name()));
```